sources/packages/backend/workflow/BPMN/camunda-8/assessment-gateway.bpmn

```diff
@@ -285,6 +298,8 @@
         <zeebe:ioMapping>
           <zeebe:input source="=studentDataTaxReturnIncome" target="reportedIncome" />
           <zeebe:input source="=null" target="supportingUserId" />
+          <zeebe:input source="=append(parentSubprocesses, &#34;studentIncomeVerificationSubprocess&#34;)" target="parentSubprocesses" />
+          <zeebe:output source="=&#34;studentIncomeVerificationSubprocess&#34;" target="studentIncomeVerificationSubprocess" />
```

Comment:
I liked very much this way of having output variables to validate the passthrough of sub process ❤️

But I believe it must become our coding standard to create sub processes(to be in wiki) as this needs to consistently followed for all workflow with subprocess.